src/black/nodes.py

```diff
@@ -529,7 +529,7 @@ def is_docstring(leaf: Leaf) -> bool:
         return False
 
     prefix = get_string_prefix(leaf.value)
-    if "b" in prefix or "B" in prefix:
+    if "b" in prefix or "B" in prefix or "f" in prefix or "F" in prefix:
```

Comment:
Might be slightly cleaner?

```suggestion
    if set(prefix).intersection("bBfF"):
```